documentation/docs/apis/autopeering.md

```diff
@@ -1,3 +1,15 @@
+---
+description: The peering API allows retrieving basic information about autopeering using the /autopeering/neighbors endpoint or the GetAutopeeringNeighbors() function in the client lib.
+image: replaceWithGoShimmerLogo
```

Comment:
we don't have the logo yet, right?

Reply:
I've added the wiki icon just now, but let's wait and see if there's a btter